### PR TITLE
Tweaks to the runtime in order to align with the current templates

### PR DESCRIPTION
**Pull request by javiercn**
Runtime changes for templates

### CODE REVIEW

File: samples/IdentityOIDCWebApplicationSample/Areas/Identity/Controllers/ApplicationAuthorizationController.cs [hunk 1]
```diff
@@ -10,16 +10,16 @@
 
 namespace IdentityOIDCWebApplicationSample.Identity.Controllers
 {
-    [Area("IdentityService")]
-    public class IdentityServiceController : Controller
+    [Area("Identity")]
+    public class ApplicationAuthorizationController : Controller
```
Comment (HaoK) [inline samples/IdentityOIDCWebApplicationSample/Areas/Identity/Controllers/ApplicationAuthorizationController.cs]: Why not just AuthorizationController?

File: samples/IdentityOIDCWebApplicationSample/Areas/Identity/Controllers/ConfigurationMetadataController.cs [hunk 2]
```diff
@@ -4,38 +4,38 @@
 
 namespace IdentityOIDCWebApplicationSample.Identity.Controllers
 {
-    [Area("IdentityService")]
-    public class IdentityServiceConfigurationController : Controller
+    [Area("Identity")]
+    public class ConfigurationMetadataController : Controller
```
Comment (HaoK) [inline samples/IdentityOIDCWebApplicationSample/Areas/Identity/Controllers/ConfigurationMetadataController.cs]: Is either Configuration or Metadata enough to mean effectively the same thing?

File: samples/IdentityOIDCWebApplicationSample/Areas/Identity/Extensions/IdentityServiceExtensions.cs [hunk 3]
```diff
@@ -13,29 +13,24 @@
 
 namespace Microsoft.AspNetCore.Identity.Service.Extensions
 {
-    public static class IdentityServiceExtensions
+    public static class IdentityApplicationExtensions
     {
-        public static IIdentityServiceBuilder AddClientInfoBinding(this IIdentityServiceBuilder builder)
+        public static IIdentityServiceBuilder AddClientExtensions(this IIdentityServiceBuilder builder)
```
Comment (HaoK) [inline samples/IdentityOIDCWebApplicationSample/Areas/Identity/Extensions/IdentityServiceExtensions.cs]: I know we discussed Identity.Service => Identity.ClientApplication(s?), but maybe Identity.Client would mean roughly same thing, so would end up being mostly Application => Client which I think has the same semantics but is shorter

Comment (HaoK) [inline samples/IdentityOIDCWebApplicationSample/Areas/Identity/Extensions/IdentityServiceExtensions.cs]: something to consider for later anyways

File: samples/IdentityOIDCWebApplicationSample/IdentityOIDCWebApplicationSample.csproj [hunk 4]
```diff
@@ -7,12 +7,17 @@
     <MvcRazorCompileOnPublish>true</MvcRazorCompileOnPublish>
     <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wp8+wpa81;</PackageTargetFallback>
     <UserSecretsId>aspnet-IdentityWebApplicationSample-71B323EE-A696-4709-9EA5-C93EFCD9E6FB</UserSecretsId>
+    <TypeScriptToolsVersion>2.3</TypeScriptToolsVersion>
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Remove="Areas\IdentityService\**" />
```
Comment (Eilon) [inline samples/IdentityOIDCWebApplicationSample/IdentityOIDCWebApplicationSample.csproj]: Remove?

**Comment by smitpatel** [comment]
the branch rel/2.0.0-preview2 has been deleted and converted to tag. This PR becomes stale and auto-closed by GitHub. If the work is required to be merged in then rebase on current dev/release and create a new PR.

**Pull request closed without merging**